packages/shared/src/models/bootc.ts

```diff
@@ -29,3 +29,5 @@ export interface BootcBuildInfo {
 }
 
 export type BootcBuildStatus = 'running' | 'creating' | 'success' | 'error' | 'lost' | 'deleting';
+
+export const BOOTC_HISTORY_FILENAME = 'history.json';
```

Comment:
Why is this in /shared? I don't see it used in frontend, and seems like something that should be kept within backend (or if it was shared, then frontend could poll the file itself...).

Reply:
Used in both history.ts and historyNotifier.ts so shared was going to be the best place for it, as well as if we had to do anything in the frontend.

Reply:
I would prefer it stayed in backend - odd to tell the frontend where a file is, but then expect it to use backend API and notifier. Better that it knows nothing about where/how the history is saved.

Reply:
Fixed.